server.js

```diff
@@ -19,6 +19,14 @@ app.use(express.json());
 app.use(express.urlencoded({ extended: true }));
 app.use(cors());
 
+// helpers
+
+const axiosGet = (path, response) => {
+  axios.get(`${url}${path}`, { headers })
+    .then(results => { response.send(results.data); })
+    .catch(err => console.error('Error executing Axios GET from API: ', err));
+};
```

Comment:
Sweet! I'll be adding an `axiosPOST` and an `axiosPUT` with my own PR.

Reply:
dope!